data/remnant missions.txt

```diff
@@ -493,5 +597,771 @@ mission "Remnant: Technology Available"
+			`	As you descend through the atmosphere, you do some quick hand stretches to work the stiffness out of your fingers and rehearse some of the more common phrases. As soon as <ship> is parked you exit and look around and approach a dock hand who doesn't appear to be too busy and carefully sign "Can you direct me to the port director?" He looks at you curiously for a moment and replies with exaggerated slowness. You thank him and head towards the spaceport following his directions.`
+
+			
+mission "Remnant: Learning Sign 2"
+	name "Learning Remnant Sign"
+	description `Return to Caelian to finish your first class of sign language.`
+	source "Viminal"
+	destination "Caelian"
+	to offer
+		has "license: Remnant"
+		has "Remnant: Learning Sign 1: done"
+	on offer
+		conversation
+			`Once again the director seems to be expecting you, and has a simple conversation with you about how the flight was and how your lessons are progressing. You struggle to string the sentences together, but you successfully manage to hold your own for several minutes before he directs you to place your hand on the scanner. After checking the readout he hands you another datachip as well as a stack of holographic print-outs.`
+			`	Flipping through the stack of print-outs, you note that each one of them has a small label in the corner for random pieces of furniture and ship equipment, and when you view the hologram while moving from side to side the figure inside makes a sign that you assume to be what the label indicates.`
+			choice
+				`	(Accept them.)`
+					goto ContinueSign
+				`	(Continue your lessons some other time.)`
+					defer
+				`	(Not interested after all.)`
+					decline
+			label "ContinueSign"
+			`	Back on your ship, you find a roll of tape and stick up all the labels to the appropriate things around your ship. Now every time you look around your ship, you see figures demonstrating how to sign things. It kind of reminds you of being back in preschool. It does seem to be working, though.`
+				accept
+	
+	on complete
+		log "Have learned the basics of the Remnant sign language, and been formally granted access to their worlds after having been genetically tested on each of them."
+		conversation
+			`The last leg of the journey was spent watching some ancient show about a merchant captain making his way through the galaxy with an unlikely crew, re-enacted by a cast of Remnant who sing and sign their way through the dialogs. It seems really familiar, but you can't quite place it. You note that it seems to have a lot of subtext about the dangers of manipulation and government secrecy, especially surrounding genetics.`
+			`	You arrive at the Director's desk and offer to return the datapad to the Director, signing your thanks. She seems pleased at how much you have learned, and replies that you are welcome to keep it for as long as you need it.`
```

Comment:
Director's office, rather than director's desk.

Reply:
I suppose my description wasn't terribly clear, but I was intending these directors to be literally that - People whose desks are out in the thick of things telling people where they should be going and giving directions. Kind of front-line operations sort of place. 

I'd have said a visitor information center or something, but this is the Remnant. The Player is the first outsider to have visited them peacefully in centuries as far as we know, so it is rather unlikely they have much in the way of tourism.

Perhaps I should switch "desk" to "kiosk" ?